### DISCUSSION TITLE
Verifying customer's email

### DISCUSSION THREAD

**Discussion by winston0410** [medusajs/medusa#4330] [Feature Requests]
Reading the [documentation for customer](https://docs.medusajs.com/modules/customers/storefront/implement-customer-profiles), there seems to be no mention on verifying customer's email, and there is no column related with `verify` in `customer` table as well. 

Is this feature going to be built, or is there any reason for not building that?

**Comment by gempain** [comment]
I'd like to up this question, because it seems to me that this is a major basic feature.

**Reply by tanishq-rocks** [reply]
Hi @gempain  Any update on this?

**Comment by ashishsah1000** [comment]
I don't think that this feature exists. How team planning to add this feature? Any insights would be helpful.

**Comment by Loque18** [comment]
+1 here

**Comment by difrisul-dev1** [comment]
what is the point of having email authentication if no way of actually verifying the informed email exists?
